packages/client/src/i18n/messages/user.ts

```diff
@@ -499,6 +500,13 @@ const messagesToDefine: IUserMessages = {
       'This error messege shows when user try to input already exsisted mobile number',
     id: 'system.user.duplicateMobileError'
   },
+  duplicateUserEmailErrorMessege: {
+    defaultMessage:
+      '{email} is already used by another user. Please use a different email',
+    description:
+      'This error messege shows when user try to input already exsisted email',
```

Comment:
>'This error messege shows when user try to input already exsisted email',
"This error message shows when user tries to input an already existing email" to fix typos